src/storage.ts

```diff
@@ -52,14 +52,26 @@ export class GlobalStorage {
    * to create a more specific Uri.
    */
   getHistoryUri(...segments: string[]): vscode.Uri {
-    // Example: $USER/Library/Application Support/Code/User/globalStorage/undefined_publisher.vscode-scriptiq/v1/scriptiq_history
+    // Example: /Users/$USER/Library/Application\ Support/Code/User/globalStorage/saucelabs.vscode-scriptiq/v1/scriptiq_history/
     return vscode.Uri.joinPath(
       this.storageUri,
       'scriptiq_history',
       ...segments,
     );
   }
 
+  verifyTestIds(ids: string[]): string[] {
```

Comment:
1. A method like `deleteTestRecord` is self explanatory and doesn't have a strong need for documentation, but `verifyTestIds` does.
2. `verifyTestIds` should be either called `verifyTestIDs` to be case-consistent with memento, or memento should be changed to follow this case instead. ⬅️ ➡️

Reply:
Ah yeah. Forgot to push a commit to rename it to `verifyTestIDs` 😂